platforms/hermes-python/README.rst

```diff
@@ -485,6 +485,84 @@ end-user.
            .start()
 
 
+Dynamic Vocabulary using Entities Injection
+===========================================
+
+Please refer to the `official documentation
+<https://docs.snips.ai/articles/platform/nlu/dynamic-vocabulary>`_ for
+further information.
+
+Sometimes, you want to extend your voice assistant with new vocabulary
```

Comment:
Some use cases requires the app to be able to extend at runtime the vocabulary the voice assistant can understand.

Reply:
Well actually, *runtime* is a bit vague here. We do have to retrain + reload ASR & NLU models before seeing the injection in effect.